TrustKit/Framework/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
I think we should include the tracking as well similar to.
```	
       <key>NSPrivacyTracking</key>
	<false/>
	<key>NSPrivacyTrackingDomains</key>
	<array/>
	<key>NSPrivacyAccessedAPITypes</key>
	<array>
```

Reply:
It is probably worth having all of the compatible keys present and specify no for tracking, empty array for tracking domains, and fill out the key for privacy label.

If I'm not mistaken, if a cert pin fails, the SDK will transmit aka "collect" data and send off device. Being clear and transparent would include that information. It would probably qualify as App Privacy Label "other data".

Reply:
I have reflected on the suggestion. Thank you. (c85ade9ebe4636a22ad2664449a06591dc6055cc)